packages/desktop-client/src/components/admin/UserAccess/UserAccessRow.tsx

```
@@ -0,0 +1,143 @@
+
+    return (
+      <Row
+        height="auto"
+        style={{
+          fontSize: 13,
+          backgroundColor: backgroundFocus
+            ? theme.tableRowBackgroundHover
+            : theme.tableBackground,
+        }}
+        collapsed={true}
+        onMouseEnter={() => onHover && onHover(access.userId)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)